src/style.css

```
@@ -1 +1,30 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('./images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```

```suggestion
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```
with such selector you can set margin-right for all cases